APCAppCore/APCAppCore/DataSubstrate/Model/APCDataMigrationPolicy_Model4ToModel6.m

```diff
@@ -94,7 +93,7 @@ - (NSSet *) linkTasksToV6ScheduleMatchingV4Schedule: (id) scheduleFromV4
         NSFetchRequest *request = [APCTask requestWithPredicate: [NSPredicate predicateWithFormat: @"%K == %@ && %K == %@",
                                                                   NSStringFromSelector (@selector (taskID)),
                                                                   taskIdV6,
-                                                                  NSStringFromSelector (@selector (taskVersionNumber)),
+                                                                  @"taskVersionNumber",
```

Comment:
yeah, we already decided that migration was going to affect a small enough number of folks that we didn't take the (substantial) effort to test it still worked before submitting 1.2 so this is probably irrelevant. I shoulda looked at the filename :P